webapp/src/main/java/com/box/l10n/mojito/entity/ApplicationCacheType.java

```diff
@@ -27,6 +27,7 @@ public class ApplicationCacheType {
   @JsonView(View.IdAndName.class)
   protected short id;
 
+  @JsonView(View.Default.class)
```

Comment:
Fields without @JsonView annotation were included in the response of an annotated endpoint that uses an specific view